packages/core/src/hooks/breadcrumb/index.ts

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import humanizeString from "humanize-string";
+
+import { useResource, useRouterContext } from "@hooks";
+import { ResourceRouterParams } from "src/interfaces";
+
+export type BreadcrumbsType = {
+    label: string;
+    to?: string;
```

Comment:
```suggestion
    href?: string;
```
Instead of `to`, we can use `href`, which is more suitable for HTML standards.